### PR TITLE
Tweak toast docs

### PR DESCRIPTION
**Pull request by patrickhlauke**
- a few rewordings/tweaks
- add info about toasts with focusable/actionable controls (shouldn't autohide them)

### CODE REVIEW

File: site/content/docs/5.0/components/toasts.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ Toasts are as flexible as you need and have very little required markup. At a mi
 
 ### Live
 
-Click the button the below to show as toast (positioning with our utilities in the lower right corner) that has been hidden by default with `.hide`.
+Click the button the below to show a toast (positioned with our utilities in the lower right corner) that has been hidden by default with `.hide`.
```
Comment (mdo) [inline site/content/docs/5.0/components/toasts.md:46]:
```suggestion
Click the button below to show a toast (positioned with our utilities in the lower right corner) that has been hidden by default with `.hide`.
```

File: site/content/docs/5.0/components/toasts.md [hunk 2]
```diff
@@ -83,7 +83,7 @@ Click the button the below to show as toast (positioning with our utilities in t
 
 ### Translucent
 
-Toasts are slightly translucent, too, so they blend over whatever they might appear over.
+Toasts are slightly translucent, so they blend over whatever they might appear over.
```
Comment (mdo) [inline site/content/docs/5.0/components/toasts.md:86]:
```suggestion
Toasts are slightly translucent to blend in with what's below them.
```

File: site/content/docs/5.0/components/toasts.md (hunk 1 above)
Comment (ffoodd) [inline site/content/docs/5.0/components/toasts.md:46]: Shouldn't it bé "Click the button below"?

**Review by mdo — APPROVED** [review]
Couple copy tweaks

File: site/content/docs/5.0/components/toasts.md (hunk 1 above)
Comment (patrickhlauke) [reply site/content/docs/5.0/components/toasts.md:46]: doh, embarassing to spot one but not the other...

Comment (patrickhlauke) [reply site/content/docs/5.0/components/toasts.md]: yup, slipped me by

**Review by ffoodd — APPROVED** [review]

**Pull request merged**